sdk/agrifood/azure-verticals-agrifood-farming/README.md

```diff
@@ -32,27 +32,21 @@ Use FarmBeats client library for Python to do the following.
 
 ### Authenticate the client
 
-#### Using Azure Active Directory
+#### Using Microsoft Entra ID
 
-In order to interact with the Azure FarmBeats service, your client must present an Azure Active Directory bearer token to the service.
+In order to interact with the Azure FarmBeats service, your client must present a Microsoft Entra ID bearer token to the service.
```

Comment:
Drop "ID", per the guidance at https://learn.microsoft.com/entra/fundamentals/how-to-rename-azure-ad#automate-bulk-editing-using-custom-code.
```suggestion
In order to interact with the Azure FarmBeats service, your client must present a Microsoft Entra bearer token to the service.
```